client/ayon_core/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -24,49 +28,38 @@ class SubstanceLoadProjectMesh(load.LoaderPlugin):
     order = -10
     icon = "code-fork"
     color = "orange"
+    project_templates = []
 
-    options = [
-        qargparse.Boolean(
-            "preserve_strokes",
-            default=True,
-            help="Preserve strokes positions on mesh.\n"
-                 "(only relevant when loading into existing project)"
-        ),
-        qargparse.Boolean(
-            "import_cameras",
-            default=True,
-            help="Import cameras from the mesh file."
-        )
-    ]
+    @classmethod
+    def get_options(cls, contexts):
+        template_enum = [template["name"] for template in cls.project_templates]
+        return [
+            EnumDef("project_template",
+                    items=template_enum,
+                    default="default",
+                    label="Project Template")
+        ]
```

Comment:
I personally really wouldn't want "template" defaults to be driven like this from the settings. Instead I'd want to expose the artist to the actual settings for what resolution they need, etc. This just feels like suddenly we'll be getting templates that are just 'unclear' as to what they are actually doing.

```
2048x2048 (UDIM | Texture Set Per UV Tile | Preserve strokes)
```

Also, preserve strokes is not a studio settings template decision - it's an artist decision on how they want to propagate the a mesh **update** into their scene.